docs/setup-xdebug.md

```diff
@@ -0,0 +1,8 @@
+## Set up XDebug
+
+Run inside of the Nextcloud container:
```

Comment:
Open a new subsection to debug a script from the CLI (not through the web browser).